datasets/haalcentraal/brk/dataset.json

```diff
@@ -1,12 +1,11 @@
 {
   "type": "dataset",
-  "id": "hcbrk",
+  "id": "haalcentraal_brk",
   "title": "BRK van HaalCentraal",
   "status": "beschikbaar",
   "description": "Deze dataset maakt de HaalCentraal BRK API toegankelijk, bij gebruik van login credentials. De velden zijn gebaseerd op het schema van Haal-Centraal.",
```

Comment:
Small nitpick: uniform spelling for "HaalCentraal".

Reply:
https://www.vngrealisatie.nl/producten/haal-centraal says "Haal Centraal", so changing to that.